### ISSUE TITLE
WTF is an API?

### ISSUE THREAD

**Issue by b30wulffz** [gaearon/whatthefuck.is#176]

**Comment by Martialis39** [comment]
I hope leaving a comment is kosher - I second this (I was about to open the same issue!). In my case, I have struggled with this a lot. I know what a REST Api is, but whenever people say `This version introduces breaking changes to the API` or `This is how we came up with the API for Redux` (paraphrased statement from a talk from Mark Erikson), or `Minimal API Surface Area` (a talk title from Sebastian Markbage), I get confused. Maybe @b30wulffz can chime in if this is what they meant as well, wouldn't want to highjack this issue.

**Comment by b30wulffz** [comment]
Yeah, I get it. I just posted this topic as a thread. Details and further issues related to the same topic can be mentioned here in the comments. 

Think about this issue like an abstract class, where one can add further information to make it much more useful to the community. 

No, you are not hijacking this issue😉
